docs/VARIABLES.md

```diff
@@ -2476,6 +2476,14 @@ Default:  true
 
 ***
 
+### sso_idp_cert_path
+
+SSL certificate (full path of file on control node) of IDP Domain for SSO in C3/cli. Optional, needed when IDP server has TLS enabled with custom certificate
```

Comment:
this was not included in 7.5, right?

Reply:
oh I get it. this was removed in another pr which is raised but this branch was created from 7.5.x where those changes werenot merged and thus this got added. Will have to rerun doc.py to remove it